src/standalone-entry.tsx

```
@@ -49,86 +49,73 @@ if (rootElement) {
         console.log(`[Standalone Mock] identifyApp called with: ${appName}`);
         return Promise.resolve(true);
       },
-      on: (event: string, _callback: (...args: unknown[]) => void): void => {
-        console.log(
-          `[Standalone Mock] Event listener for "${event}" registered.`,
-        );
-        return (): void =>
-          console.log(
-            `[Standalone Mock] Event listener for "${event}" unregistered.`,
-          );
+      on: (event: string, callback: Function) => {
```

ESLint

[error] Don't use `Function` as a type. The `Function` type accepts any function-like value.
It provides no type safety when calling the function, which can be a common source of bugs.
It also accepts things like class declarations, which will throw at runtime as they will not be called with `new`.
If you are expecting the function to accept certain arguments, you should explicitly define the function shape.

(@typescript-eslint/ban-types)